packages/ui-test-locator/src/utils/locator.ts

```diff
@@ -51,23 +62,26 @@ export function locator(containerSelector, customMethods = {}) {
     })
   }
 
-  const find = async (...args) => {
-    return firstOrNull(await findAll(...args))
+  const find = async (...args: QueryArguments) => {
+    // TODO remove "as any". This will result in lots of ts typing errors
+    return firstOrNull(await findAll(...args)) as any
   }
```

Comment:
Fixing this will solce one of the last typing issues, namely that XYZLocator-s are typed to "any". I did not do in this PR because it would bloat it even more